nservicebus/operations/auditing.md

```diff
@@ -71,3 +71,15 @@ If no OverrideTimeToBeReceived is defined then:
 ## Configuring auditing
 
 partial: configuration
+
+## Filtering audit messages
+
+By default all messages are audited, to control what message types that gets audited see our [audit filter sample](/nservicebus/audit-filter/).
+
+### Filtering out individual properties
+
+Auditing works by sending an exact copy of the received message to the audit queue so filtering out individual propeties is not supported.
```

Comment:
should there be a comma between "[...] to the audit queue, so filtering out [...]"?